sealtk/core/AbstractProxyModel.cpp

```diff
@@ -110,6 +113,9 @@ bool AbstractProxyModel::lessThan(
     case core::ClassificationScoreRole:
       return left.toDouble() < right.toDouble();
 
+    case core::ClassificationTypeRole:
```

Comment:
Nit: Can you move this up to before `// Boolean comparisons`, which seems to be where string comparisons belong? (Once upon a time, `NameRole` was a straight string comparison and this would have been more obvious...)

...and I guess the other hunk should get similar treatment. (If you like, it might make sense to add `// String comparisons` also, but I can go either way with that.)

Reply:
Done